bot/src/plugins/github_issues.cr

```diff
@@ -14,6 +27,16 @@ class GithubIssues
     return unless msg.channel?
     return unless config.repositories.has_key? msg.channel.name
     issues = msg.message.scan(PATTERN)
-    msg.reply issues.map {|issue| "https://github.com/#{config.repositories[msg.channel.name]}/issues/#{issue[1]}" }.join(" | ")
+    issues_data = issues.map {|issue| get_issue config.repositories[msg.channel.name], issue[1]}
```

Comment:
Joining the `map call` to the scan would be acceptable here IMO and could get rid of the additional local variable.